examples/tutorial/tutorial_part_one.ipynb

```diff
@@ -29,17 +30,46 @@
    },
    "outputs": [],
    "source": [
-    "cifar_loc = \"/Users/crankshaw/test/path/\"\n",
+    "cifar_loc = \"\"\n",
     "%run ./download_cifar.py $cifar_loc"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## Extract the images"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we must extract the data into a format we can load. This will make use of the provided [`extract_cifar.py`](extract_cifar.py)\n",
+    "\n",
+    "This dataset has 50,000 training datapoints and 10,000 test datapoints. We don't need to use all of them to demonstrate how Clipper works. Feel free to adjust `max_train_datapoints` and `max_test_datapoints` or set them to `None` if you wish to use all the data available for training and testing. You can change these vaues and re-run this command in the future if you wish."
+   ]
```

Comment:
In the JIRA issue, you said "We default to using 10,000 training datapoints (instead of the full 50,000). With 50,000 datapionts, writing to the CSV takes a long time, as does training on all the datapoint later. Using 10000 for training yields similar accuracies and prevents some of the existing delays in the tutorial runthrough."

Can you add an aside (using the quote formatting in markdown) that explains this?